lib/plugins/aws/provider.js

```diff
@@ -1195,9 +1195,15 @@ class AwsProvider {
             vpc: { $ref: '#/definitions/awsLambdaVpcConfig' },
             vpcEndpointIds: { $ref: '#/definitions/awsCfArrayInstruction' },
             versionFunctions: { $ref: '#/definitions/awsLambdaVersioning' },
-            websocketsApiName: { type: 'string' },
```

Comment:
We should still support the old properties - removing them is a breaking change and we want to avoid that until releasing new major in the future - hence the need for deprecation first